app/Events/GadgetAmountUpdatedEvent.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Events;
+
+use App\Models\User;
+use Illuminate\Broadcasting\Channel;
+use Illuminate\Broadcasting\InteractsWithSockets;
+use Illuminate\Broadcasting\PresenceChannel;
+use Illuminate\Broadcasting\PrivateChannel;
+use Illuminate\Contracts\Broadcasting\ShouldBroadcast;
+use Illuminate\Foundation\Events\Dispatchable;
+use Illuminate\Queue\SerializesModels;
+
+class GadgetAmountUpdatedEvent extends GameEvent
+{
+    public $gadgets;
+    public $user;
+
+    public function __construct($gameId, User $user)
+    {
+        $this->user = $user;
+        $this->gadgets = $user->gadgets()->get();
+        $this->gameId = $gameId;
```

Comment:
```suggestion
        parent::__construct($gameId);
        $this->user = $user;
        $this->gadgets = $user->gadgets()->get();
```
Dit verandert niks, maar deze manier passen we wel bij andere events toe. Dus het lijkt me netjes als we dat overal doen.

Reply:
Heb ik nu aangepast bij aankomende commit